src/UserInterface/CarMap.py

```diff
@@ -180,3 +191,25 @@ def __run_async_task(self, task: Coroutine[Any, Any, None]) -> None:
         asyncio.create_task(task)
         # TODO: Log error, if the coroutine doesn't end successfully
         return
+
+    async def __emit_scoreboard(self) -> None:
+        scoreboard_before = [{"Player-Name": "", "Score": 0}]
+        while True:
+            scoreboard = self.__create_table()
+            if scoreboard is not None:
+                await self._sio.emit('update_scoreboard', scoreboard)
+                if scoreboard != scoreboard_before:
+                    logger.info(scoreboard)
+                    scoreboard_before = scoreboard
+
+            await self._sio.sleep(1)
+
+    def __create_table(self) -> List[Dict[str, int]]:
+        """
+        Creates table from all current players with their respective nicknames and scores.
+
+        """
+        drivers = self._environment_manager.get_drivers()
+        if len(drivers) != 0:
+            return [{"Player-Name": d.get_nickname(), "Score": d.get_score()} for d in drivers]
+        return None
```

Comment:
Only active drivers should be displayed on the scoreboard. Or at least inactive players should be cleared from the drivers list after a certain amount of time.

Reply:
Good point, although this timeframe schouldn't be too short. Otherwise the scoreboard would be useless to compare yourself to others

Reply:
As long as some clearing mechanism is implemented and the application does not need to do a restart to clear the scoreboard it will be fine by me.